server/controllers/color.controller.js

```diff
@@ -0,0 +1,170 @@
+"use strict";
+const helpers = require('../../lib/helpers.js');
+const decorate = helpers.decorate;
+const metadata = helpers.metadata;
+const param = helpers.param;
+
+const repository = require("@loopback/repository");
+const rest = require("@loopback/rest");
+const models = require("../models");
+const repositories = require("../repositories");
+let ColorController = class ColorController {
```

Comment:
Let's simplify class definitions - same comment applies to all other places where we are defining a new class.

```suggestion
class ColorController {
```